docs/content/customization/generic/global-mirror.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "Global Image Registry Mirror"
++++
+
+Add containerd image registry mirror configuration to all Nodes in the cluster.
+
+When the `globalImageRegistryMirror` variable is set, `files` with configurations for [Containerd default mirror](https://github.com/containerd/containerd/blob/main/docs/hosts.md#setup-default-mirror-for-all-registries) will be added.
+
+This customization will be available when the
+[provider-specific cluster configuration patch]({{< ref "..">}}) is included in the `ClusterClass`.
+
+## Example
+
+To provide image registry mirror with CA certificate, specify the following configuration:
+
+If your registry mirror requires self signed CA certifate, create a Kubernetes Secret with keys for `ca.crt`:
+
+```shell
+kubectl create secret generic my-mirror-ca-cert-secret \
+  --from-file=ca.crt=registry-ca.crt
+```
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: <NAME>
+spec:
+  topology:
+    variables:
+      - name: clusterConfig
+        value:
```

Comment:
Can you please include here whatever is needed for `imageRegistries` for completeness. This example can include the `secretRef` and the ECR can just be the `url.

Reply:
fixed. 9014ba63f9e26c93683d932ce6df6ca77664ca8a